base/docs/helpdb/Base.jl

```diff
@@ -644,9 +644,9 @@ julia> exp10(0.2)
 exp10
 
 """
-    &(x, y)
+    x & y
```

Comment:
Should we document all of the binary operators in this form?

Maybe, in the old RST format, we needed to put the function name first in order for the manual-generator to find it, but that shouldn't be necessary any more.